woocommerce/woocommerce
# [e2e tests] New test that checks a customer can comment on a post

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce-quality/issues/723 .
Added a new spec `wordpress-post.spec.js` with a test to cover the commenting on a post by a customer.  



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Locally, execute the new test:
`pnpm test:e2e-pw ./tests/e2e-pw/tests/shopper/wordpress-post.spec.js`

2. Make sure the CI passes



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
